admin/orders.php

```diff
@@ -15,6 +15,7 @@
 $quick_view_popover_enabled = false;
 $includeAttributesInProductDetailRows = true;
 $show_product_tax = true;
+$show_zone_info = true;
```

Comment:
This should be at least a pseudo-configuration definition so that stores that don't want the display don't need to 'carve up' this core-file.  Other options would be to add a notification to enable store-specific overrides.

Reply:
OK adding a comment and facilitating using an override file in extra_configures.